src/components/button/button.tsx

```diff
@@ -12,10 +12,14 @@ export type ButtonProps = {
    * Pass properties to button element directly
    */
   buttonProps?: JSX.IntrinsicElements['button'];
+  /**
+   * Define aria-label for voiceover to read.
+   */
+  ariaLabel?: string;
```

Comment:
This already exists in the `buttonProps` I learned a week ago 😄

Reply:
Aah! I was not aware of that either. Good to know!